tools/detect_conflicts.php

```diff
@@ -583,31 +590,35 @@ function findConflict($conflict, $conflicts)
  */
 function detectIgnoreColumns($instruments, $confirm)
 {
+    global $lorisInstance;
     $instrumentFields = [];
 
     foreach ($instruments as $instrument) {
         $file = "../project/instruments/NDB_BVL_Instrument_$instrument.class.inc";
         if (file_exists($file)) {
             include_once $file;
-            $commentids      = getCommentIDs($instrument, null);
-            $instance        =& NDB_BVL_Instrument::factory(
-                $instrument,
-                $commentids[0]['CommentID'],
-                null
-            );
-            $DDEIgnoreFields = $instance->_doubleDataEntryDiffIgnoreColumns;
-
-            if ($DDEIgnoreFields != null) {
-                foreach ($DDEIgnoreFields as $key => $DDEField) {
-                    $instrumentFields = array_merge(
-                        $instrumentFields,
-                        [$DDEField => $instrument]
-                    );
+            $commentids = getCommentIDs($instrument, null)->getIterator();
+
+            foreach ($commentids as $cid) {
+                $instance        = NDB_BVL_Instrument::factory(
+                    $lorisInstance,
+                    $instrument,
+                    $cid['CommentID']
```

Comment:
Sorry! I meant using just $cid
```suggestion
                    $cid
```

Reply:
No, `$cid` is an array so this would return a type error
<img width="405" alt="Screenshot 2024-12-11 at 12 26 15 PM" src="https://github.com/user-attachments/assets/af40f4d8-ed60-42e3-86de-3aa87df9a086" />

Reply:
Oh sorry about that! I thought it was just an array of the CommentIDs